client/ayon_zbrush/hooks/pre_subprocess_no_window.py

```diff
@@ -0,0 +1,21 @@
+import subprocess
+from ayon_applications import PreLaunchHook, LaunchTypes
+
+
+class LaunchServerNoWindow(PreLaunchHook):
+    """Specifically for Zbrush to make the AYON tools launching faster
```

Comment:
Do we know WHY it's faster? if so, would be nice to add in the docstring

Reply:
Nope.

Reply:
So it's not the `stdout` and `stderr` not being used (set to None) making it faster?

Reply:
I don't know, you can try 🙂 . I don't have zbrush, and don't have spare time to debug.

Reply:
Don't have a running zbrush copy either - fine with the fix myself, was just curious.

Reply:
Created issue for that https://github.com/ynput/ayon-zbrush/issues/12

Reply:
> Don't have a running zbrush copy either - fine with the fix myself, was just curious.

It only happened when we launch with the `ayon_console.exe`. It works okay when we launch with `ayon.exe`

Reply:
If that's the case I'm pretty sure it's the `stdout` and `stderr` that fixes the slowness. Thanks for reporting back!